config/karma.base.js

```diff
@@ -101,7 +101,7 @@ const config = {
   singleRun: false,
 
   client: {
-    mocha: {timeout: 20000, retries: 3},
+    mocha: { timeout: 20000, retries: 3 },
```

Comment:
Can you a) make sure you are in sync with master and b) run `yarn` from the repo root?

This might get rid of this style update.

Reply:
Done.